Author: Pino Toscano <toscano.pino@tiscali.it>
Description: Add proper Hurd-specific _IOT defines for the video1394_mmap and
 video1394_wait structs.
---
 encodedv/dvconnect.c |    7 +++++++
 1 file changed, 7 insertions(+)

--- libdv.orig/encodedv/dvconnect.c
+++ libdv/encodedv/dvconnect.c
@@ -117,6 +117,13 @@ struct video1394_wait
 #define VIDEO1394_LISTEN_POLL_BUFFER	\
 	_IOWR('#', 0x18, struct video1394_wait)
 
+#ifdef __GNU__
+#define _IOT_video1394_mmap			\
+	_IOT(_IOTS(int), 1, _IOTS(unsigned int), 7, 0, 0)
+#define _IOT_video1394_wait			\
+	_IOT(_IOTS(unsigned int), 2, _IOTS(struct timeval), 1, 0, 0)
+#endif
+
 static int cap_start_frame = 0;
 static int cap_num_frames = 0xfffffff;
 static int cap_verbose_mode;
